### DISCUSSION TITLE
When I create the executable file via pyinstaller I get an error.

### DISCUSSION THREAD

**Discussion by haomonic9** [rdkit/rdkit#4307] [Q&A]
Hello, all

I'm having this issue. When I create an executable file with **pyinstaller** the code using the **rdMolStandardize** package in rdkit, there is no error on my computer, but when I run it on another computer, I get the following error:

![error2](https://user-images.githubusercontent.com/78338032/124548424-a8822a80-de68-11eb-83bf-725e22f03cee.png)

The problem seems to be that "**.../MolStandardize/normalizations.txt**" is not included in the executable file and is loaded from the local path on my computer. How can I fix this?

The version of rdkit I used is 2020.09.1.

**Comment by greglandrum** [comment]
The current (and previous) releases of the RDKit require some parameter files which are loaded at run time in order to be able to do particular operations. One of those operations is standardizing molecules. 
I don't know how to use pyinstaller, but if you can't get it `pyinstaller` to include those parameter files in the packages it creates, then you won't be able to use that functionality in your wrapped executables.

The files in question are in `$RDBASE/Data/MolStandardize`
